### PR TITLE
Add list of required IAM permissions needed for Dallinger

### PR DESCRIPTION
**Pull request by alecpm**

### CODE REVIEW

**Review by jessesnyder — APPROVED** [review]
This is probably good enough. I feel like a picture is worth a thousand words when it comes to AWS configuration, and I thought I had a screenshot of the exact right thing, but it doesn't include S3 access(!). Do you think it would be worth getting an image of the correct config?

AWS_IAMGroupWithSNSPerms

**Comment by alecpm** [comment]
@jessesnyder Yeah, I think that could be worthwhile, but we'd probably want to make sure we blur out any account details. I think I've found a better AWS doc link that captures IAM user creation, permission setting, and access key generation. It doesn't have screenshots, but it might be sufficient. https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html

**Comment by jessesnyder** [comment]
> @jessesnyder Yeah, I think that could be worthwhile, but we'd probably want to make sure we blur out any account details. I think I've found a better AWS doc link that captures IAM user creation, permission setting, and access key generation. It doesn't have screenshots, but it might be sufficient. https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html

Yeah, that doc's better than the one we currently link to, and I agree that's good enough.

**Comment by alecpm** [comment]
I've added a screenshot.

File: docs/source/aws_etc_keys.rst [hunk 1]
```diff
@@ -58,10 +58,25 @@ successfully used AWS.
...
+<https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_create.html>`__.
+
+If you are using an IAM user instead of an AWS root account, then you will
+need to ensure the IAM user is granted the following permissions::
+
+    AmazonS3FullAccess
+    AmazonMechanicalTurkFullAccess
+    AmazonSNSFullAccess
+
+You may want to assign these permissinons by creating a Dallinger Group in
```
Comment (jessesnyder) [inline docs/source/aws_etc_keys.rst:73]:
```suggestion
You may want to assign these permissions by creating a Dallinger Group in
```

**Pull request merged**